app/components/progress/index.js

```diff
@@ -28,8 +28,8 @@ export class ActProgress extends Component {
     const {index, length} = this.props;
     return (
       <View padder style={styles.progress}>
-        <Text style={styles.progressValue}>{`${index+1}/${length}`}</Text>
-        <Progress.Bar color={styles.progress.color} style={{flexGrow: 1}} progress={(index + 1) / length} width={null} height={12}/>
+        {/* <Text style={styles.progressValue}>{`${index+1}/${length}`}</Text> */}
```

Comment:
I tend to favour deleting comments unless I know I'm going to want them in the next commit. There's always the git log if we regret deleting something.